tests/apps/perf/workflowsapp/app.py

```diff
@@ -153,38 +162,40 @@ def run_workflow(run_id):
         })
 
     with DaprClient() as d:
-        print("==========Start Workflow:==========")
         try:
+            print(f"{datetime.now():%Y-%m-%d %H:%M:%S.%f} [{run_id}] starting workflow")
             start_resp = d.start_workflow(workflow_component=workflowComponent,
                         workflow_name=workflowName, input=input, workflow_options=workflowOptions)
-            print(f"started workflow with instance_id {start_resp.instance_id}")
+            print(f"{datetime.now():%Y-%m-%d %H:%M:%S.%f} [{run_id}] started workflow with instance_id {start_resp.instance_id}")
             instance_id = start_resp.instance_id
         except DaprInternalError as e:
-            print(f"error starting workflow: {e.message}")
-        
+            print(f"{datetime.now():%Y-%m-%d %H:%M:%S.%f} [{run_id}] error starting workflow: {e.message}")
+
+        sleep(0.5)
```

Comment:
Added this to make the test more reliable in the results.

The previous operation ("start_workflow") starts the workflow by creating a reminder, so that's async. The next operation, "wait_for_workflow_completion", does a poll every 4s. So, if the first poll happens before the reminder has been executed, we waste 4s for no reason, impacting the outcome of the perf test.